Contest_API.md

```diff
@@ -610,13 +610,32 @@ The following endpoint is associated with API information:
 
 Properties of version object:
 
+| Name         | Type              | Description
+| :----------- | :---------------- | :----------
+| version      | string            | Version of the API. For this version must be the string `draft`. Will be of the form `<yyyy>-<mm>`, `<yyyy>-<mm>-draft`, or simply `draft`.
+| version\_url | string            | Link to documentation for this version of the API.
+| provider     | provider object ? | Information about the data provider
+
+Properties of the provider object:
+
 | Name         | Type            | Description
 | :----------- | :-------------- | :----------
-| version      | string          | Version of the API. For this version must be the string `draft`. Will be of the form `<yyyy>-<mm>`, `<yyyy>-<mm>-draft`, or simply `draft`.
-| version\_url | string          | Link to documentation for this version of the API.
 | name         | string ?        | Name of this data provider.
+| version      | string ?        | Provider's application version string
+| build\_date  | TIME ?          | TIME provider's application was built
+| type         | string ?        | See Known Provider Types table below
```

Comment:
I'm not totally opposed to this, but why would we provide this and how would we expect clients to behave differently? It just seems like extraneous information. Does it matter to the client if you're directly connected to a CCS or there is a proxy in the middle?

CMS would need an entry too, maybe other proxies.

Reply:
It was decided `type `is not necessary since the `name `property should be enough.
`type `property removed, as well as the **Known Provider Types** table.